plugin/slime.vim

```diff
@@ -44,26 +44,34 @@ endfunction
 " Tmux
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
+function! s:TmuxSocket()
+  " The socket path is the first value in the comma-separated list of $TMUX.
+  return split($TMUX, ',')[0]
```

Comment:
The presence of the TMUX environment variable assumes that we are running VIM inside TMUX. That's not necessarily the case…